src/usecases/handle-figma-file-update-event-use-case.ts

```diff
@@ -60,9 +60,37 @@ export const handleFigmaFileUpdateEventUseCase = {
 
 		if (!designs.length) return;
 
-		await jiraService.submitDesigns(
-			designs.map((design) => ({ design })),
-			connectInstallation,
-		);
+		// Update the timestamp for all the designs that updated their dev status
+		await Promise.all([
+			await jiraService.submitDesigns(
```

Comment:
@Roystbeef I think I'm missing something here. Can you provide some detail about how this process works?

From what I can tell:
* `figmaService.getAvailableDesignsFromSameFile` returns the node with the stored `devStatusLastModified` on line 47 above
* Then here you send the design to Jira and update the `devStatusLastModified` at the same time

Wouldn't this send a stale value to Jira?

Reply:
Thank you for the description in the summary. I understand how this works now. My concern with this approach is it's a lot of complexity to workaround an issue that is better solved in Figmas API.

Reply:
> My concern with this approach is it's a lot of complexity to workaround an issue that is better solved in Figmas API.

Yeah I agree that having Figma's API handle this is most definitely the better long-term solution. My understanding of the situation is the current `lastModified` value we currently expose for nodes just surfaces a value that previously was only used privately for a different set of use cases. There's a few approaches we could take for this API going forward (eg. building it with the REST API in mind so that devstatus changes (among others) are reflected in the timestamp).

Unfortunately, IMO this is the only feasible solution to getting this ready by GA and IMO feels better scoped since any tech debt we're introducing is only on the relevant integration :/

Reply:
Sounds good @Roystbeef and we're happy to defer to you as maintainer of the app however please consider this issue was not raised by customers but only found by us while testing. You could defer until Figmas API can be changed, keeping this workaround in your back pocket in case you receive feedback from customers.